binding/python/src/bindings/slogger.rs

```diff
@@ -8,6 +8,30 @@ use sloggers::{
     Build,
 };
 
+#[pyclass(name = "Level")]
+#[derive(Clone, Copy, Debug)]
+pub enum PyLevel {
```

Comment:
Let's use the type names as they are used in slog by default. (`Severity` instead of `Level`.)